app/src/main/res/values/strings.xml

```diff
@@ -58,6 +58,8 @@
     <string name="storage_fragment_restore_title">Where to find your backups?</string>
     <string name="storage_fragment_warning">People with access to your storage location can learn which apps you use, but do not get access to the apps\' data.</string>
     <string name="storage_fragment_warning_delete">Existing backups in this location will be deleted.</string>
+    <string name="storage_user_selected_location_title">User-chosen location</string>
+    <string name="storage_user_selected_location_summary">Chosen using the folder browser</string>
```

Comment:
If this is never shown to the user, we may not need those extra strings.

Reply:
I pushed a new commit that removes the summary as the title does seem to get shown to the user.